core/modules/wsched/ChunkTaskList.cc

```diff
@@ -0,0 +1,387 @@
+// -*- LSST-C++ -*-
```

Comment:
A creating a ticket to handle it in the future would probably be good. There's a significant amount of flux still in the implementation, but the basic design is the same. Group tasks by chunkId and table so that several tasks can run while only loading the table once.